config/config_test.go

```diff
@@ -3150,3 +3178,44 @@ func TestTCF2FeatureOneVendorException(t *testing.T) {
 		assert.Equal(t, tt.wantIsVendorException, value, tt.description)
 	}
 }
+
+func TestAccountPriceFloors_validate(t *testing.T) {
+	type fields struct {
+		Enabled                bool
+		EnforceFloorRate       int
+		AdjustForBidAdjustment bool
+		EnforceDealFloors      bool
+		UseDynamicData         bool
+	}
+	type args struct {
+		errs []error
+	}
+	tests := []struct {
+		name string
```

Comment:
Super nitpick: can you call this field `description` instead to match recent tests?

Reply:
updated